packages/roosterjs-editor-api/lib/utils/execCommand.ts

```diff
@@ -32,6 +38,22 @@ export default function execCommand(editor: IEditor, command: DocumentCommand) {
             });
         }
     } else {
-        editor.addUndoSnapshot(formatter, ChangeSource.Format);
+        editor.addUndoSnapshot(() => {
+            let tempRange: Range;
+            selection.ranges.forEach(range => {
+                if (selection.type == SelectionRangeTypes.TableSelection) {
+                    editor.select(range);
+                }
+
+                formatter();
+
+                tempRange = range;
+            });
+
+            if (tempRange && selection.type == SelectionRangeTypes.TableSelection) {
+                tempRange.collapse();
+                editor.select(tempRange);
+            }

```

Comment:
what is this part for?

Reply:
Since table selection is not using actual browser range if we keep this, the last range of the table is going to be selected.
I think it could be a little confusing to the user if they see it, but we can remove it.

![image](https://user-images.githubusercontent.com/8291124/151870273-e156e663-944a-40c5-b65f-8931a23a163a.png)